testing/side-by-side.tsx

```diff
@@ -39,15 +40,18 @@ const SideBySide = ({
 const styles = {
     sideBySide: {
         display: "flex",
+        flexWrap: "wrap",
         flexDirection: "row",
     },
     leftPanel: {
         paddingRight: "30px",
-        flexGrow: 1,
+        flexBasis: "400px",
+        margin: Spacing.medium_16,
     },
     rightPanel: {
         flexGrow: 1,
         maxWidth: "50%",
+        flexBasis: "400px",
```

Comment:
I wonder if this `400px` magic number should be extracted to a constant. I suspect there are other places where we reference 400px, too.

Reply:
I found a few places that had a 400 constant already.  I've refactored to use one of them.  I think 400 is a graph constant mostly, but for our stories I think it mostly makes sense for now.